src/helm/benchmark/presentation/run_display.py

```diff
@@ -229,6 +231,15 @@ def write_run_display_json(run_path: str, run_spec: RunSpec, schema: Schema, ski
     predictions: List[DisplayPrediction] = []
     requests: List[DisplayRequest] = []
 
+    if validity_check:
+        validity_path: Optional[str] = hf_hub_download(
+            repo_id="stair-lab/helm_display_validity",
+            repo_type="dataset",
+            filename="validity.pkl"
```

Comment:
Pickle is not secure, so this allows an attacker who gains control of `stair-lab/helm_display_validity` repository to run arbitrary Python code on HELM users' machines. Please use an alternative serialization format instead such as CSV, JSON, JSONL or Parquet. See the [datasets documention](https://huggingface.co/docs/datasets/en/repository_structure) for more suggestions.